pkg/api/validation/validation.go

```diff
@@ -52,6 +52,7 @@ const fieldImmutableErrorMsg string = `field is immutable`
 const isNotIntegerErrorMsg string = `must be an integer`
 
 var pdPartitionErrorMsg string = validation.InclusiveRangeError(1, 255)
+var volumeModeErrorMsg string = validation.InclusiveRangeError(0, 0777)
```

Comment:
On Wed, Aug 03, 2016 at 08:53:08PM -0700, Tim Hockin wrote:
> > @@ -52,6 +52,7 @@ const fieldImmutableErrorMsg string = `field is immutable`
> >  const isNotIntegerErrorMsg string = `must be an integer`
> >  
> >  var pdPartitionErrorMsg string = validation.InclusiveRangeError(1, 255)
> > +var volumeModeErrorMsg string = validation.InclusiveRangeError(0, 0777)
> 
> Still to be resolved

Yes, I didn't forget about this. More validations are also missing :-D